requirements/run.txt

```diff
@@ -1,6 +1,6 @@
-numpy < 2.0.0  # Need to fix deprecations before 2.0.0 compat
-wxpython
-diffpy.pdffit2
-diffpy.structure
-diffpy.utils
-matplotlib-base
+numpy
```

Comment:
Removed pinning since the latest release on conda-forge (diffpy.structure, diffpy.pdffit2 have no numpy pinning)

Reply:
In `proproject.toml`, however, still maintained as discussed with the wx problem. 

```
        'Programming Language :: Python :: 3.10',
        'Programming Language :: Python :: 3.11',
```